jmslib/src/main/java/com/redhat/mqe/lib/ConnectorClient.java

```diff
@@ -77,7 +77,7 @@ private void startConnections() {
             try {
                 connection.start();
                 connectionsOpened++;
-            } catch (JMSException e) {
+            } catch (JMSException | NullPointerException e) {
```

Comment:
Tohle je dost prasarna. Maskuje to NPE kvuli programatorske chybe.

Nahradil bych `System.exit(1);` za `throw new JmsMessagingException(...)` a tady bych ju chytal a strcil bych ji do `exceptions`.